src/utils/constants.ts

```diff
@@ -128,4 +130,6 @@ export default {
   PAGE_TITLE_CREATE: 'New Application | Application Manager',
   PAGE_TITLE_APPLICATION: 'Application %1 | Application Manager',
   PAGE_TITLE_REVIEW: 'Review %1 | Application Manager',
+  DEFAULT_TEMPLATE_CATEGORY_TITLE: 'General',
+  DEFAULT_TEMPLATE_CATEGORY_ICON: 'globe',
```

Comment:
I think we should keep this file for just "translatable" strings, maybe make a new one for other constants? (Cos we don't want some hapless translator accidentally translating the word "globe" for example)

Reply:
I was thinking this can be changed by the admin user, but you are right, this potentially could be translated by accident. not sure what to do

Reply:
Let's change this file to "language strings" and make a new constants.ts?